### DISCUSSION TITLE
Bulk Download

### DISCUSSION THREAD

**Discussion by Binary-ShadowMonarch** [pystardust/ani-cli#1326] [Q&A] [answered]
How can I download whole season of anime. For now I am just downloading one episode at at time. HELP

**Comment by Dev81078** [comment]
एक समय की बात है, एक छोटे से गाँव में जहाँ पहाड़ियों की चोटियों से घिरा हुआ और जंगलों की सरसराहट महसूस होती थी, वहाँ एक उत्सुक बच्चा रहता था जिसका नाम आदित्य था। जैसे सूरज की तरह व्याप्त आंखों और चारों ओर की प्रकृति से भरी हुई दिल वाला आदित्य अपने घर के आस-पास के प्राकृतिक रहस्यों का अन्वेषण करने में अपने दिन बिताता था।

प्रत्येक सुबह, सूरज की पूरी तरह से उगती हुई तब तक, आदित्य अपने कमरे से बाहर टिपटोकरते हुए उठ जाता, दिन को स्वागत करने के लिए उत्सुक होता। एक सिरी में झाड़ियों के बीच में, वह अपनी विश्वसनीय कुत्ते, गहरे विचारों के साथ निकलता था। एक साथ, वे जादुई जंगल में भटकते, जहाँ जंगली फूलों की नृत्य करती हुई रहस्यमय खुलियों की खोज करते थे और गुप्त नदियों में बसी प्राचीन कहानियों को सुनते थे।

लेकिन आदित्य की उत्सुकता केवल प्राकृतिक विश्व की सीमा तक ही सीमित नहीं थी। वह कहानियों का प्रेमी भी था, और हर रात, अपने चादर के नीचे निवास करते हुए, वह अपनी पसंदीदा किताबों की पन्नों में खो जाता। बहादुर योद्धाओं और महान ड्रैगन्स की कहानियों से लेकर दूर-दूर के देशों और साहसी प्रशिक्षणों की कहानियों तक, आदित्य की कल्पना की कोई सीमा नहीं थी।

**Comment by mrfluffy-dev** [comment]
Another demon summoner 😔

**Reply by CoolnsX** [reply]
@mrfluffy-dev 
lmao, it seems it's an advertisement

**Comment by mrfluffy-dev** [comment, marked as answer]
> How can I download whole season of anime. For now I am just downloading one episode at at time. HELP

anyway I think something like this will work 

`ani-cli -d -r  "1 -1" cyberpunk edgerunners`

`ani-cli` base command 

`-d` download 

`-r` range

`"1 -1"` from episode 1 to episode -1 (last episode) 

`cyberpunk edgerunners` anime to download

**Reply by Binary-ShadowMonarch** [reply]
for some reason its showing download error with some network issues but I think this is it if the error is fixed. ani-cli might not be it for downloading whole season. I can do it somewhere else😂.
